src/NextGen/DonationForm/ViewModels/DonationConfirmationReceiptViewModel.php

```diff
@@ -0,0 +1,214 @@
+<?php
+
+namespace Give\NextGen\DonationForm\ViewModels;
+
+use Give\Donations\Models\Donation;
+use Give\Framework\EnqueueScript;
+use Give\NextGen\DonationForm\FormDesigns\DeveloperFormDesign\DeveloperFormDesign;
+use Give\NextGen\DonationForm\Models\DonationForm;
+use Give\NextGen\DonationForm\Repositories\DonationFormRepository;
+use Give\NextGen\Framework\FormDesigns\Registrars\FormDesignRegistrar;
+use Give\NextGen\Framework\Receipts\DonationReceipt;
+use Give\NextGen\Framework\Receipts\DonationReceiptBuilder;
+
+/**
+ * @unreleased
+ */
+class DonationConfirmationReceiptViewModel
+{
+    /**
+     * @var Donation
+     */
+    public $donation;
+
+    /**
+     * @unreleased
+     */
+    public function __construct(Donation $donation)
+    {
+        $this->donation = $donation;
+    }
+
+    /**
+     * @unreleased
+     */
+    public function getDonationForm(): DonationForm
+    {
+        return DonationForm::find($this->donation->formId);
+    }
+
+    /**
+     * @unreleased
+     */
+    public function getReceipt(): DonationReceipt
+    {
+        $receipt = new DonationReceipt($this->donation);
+        $receipt->settings->addSetting(
+            'heading',
+            __("Hey {$this->donation->firstName}, thanks for your donation!", 'give')
+        );
+        $receipt->settings->addSetting(
+            'description',
+            __(
+                "{$this->donation->firstName}, your contribution means a lot and will be put to good use in making a difference. We’ve sent your donation receipt to {$this->donation->email}.",
```

Comment:
Are these string values customizable somewhere or are we using these hard-coded strings?

Reply:
This is a parity setting that will come from the form builder receipt settings.